src/components/simple/heading.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Heading as DecanterHeading } from 'decanter-react'
+import { smallPaddingTop, smallPaddingBottom } from "../../utilities/dataSource";
+import { dcnb } from "cnbuilder";
+
+const Heading = ({blok: {title, fontSize, headingLevel, spacingBottom, spacingTop}}) => {
+  const paddingTop = smallPaddingTop[spacingTop] ?? smallPaddingTop.lg;
+  const paddingBottom = smallPaddingBottom[spacingBottom] ?? smallPaddingBottom.lg;
+
+  return (
+    <DecanterHeading
+      level={headingLevel ?? 2}
```

Comment:
```suggestion
      level={parseInt(headingLevel, 10) || 2}
```
Heading wants an integer.